docs/stubs.rst

```diff
@@ -170,6 +171,35 @@ specified in ``__all__`` are imported::
 
 Type checkers support cyclic imports in stub files.
 
+Built-in Generics
+-----------------
+
+PEP 585 [#pep585]_ built-in generics are generally supported, with
+the following exceptions:
+
+* Built-in generics don't work in type aliases.
```

Comment:
To help this stay up to date, it would be useful to link to the appropriate mypy issues.

Reply:
Good idea, although I would propose to link to the typeshed tracking issues as they give a bit better overview for all type checkers.